recipe/build.sh

```diff
@@ -19,7 +19,7 @@ eval export ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib"
 # Build LAPACK.
 # Enable threading. This can be controlled to a certain number by
 # setting OPENBLAS_NUM_THREADS before loading the library.
-make QUIET_MAKE=1 DYNAMIC_ARCH=1 BINARY=${ARCH} NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 CFLAGS="${CF}"
+make QUIET_MAKE=1 DYNAMIC_ARCH=1 BINARY=${ARCH} NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 CFLAGS="${CF} -fPIC" FCOMMON_OPT="-fPIC -frecursive"
```

Comment:
Saw some flags for `gfortran` were being dropped when removing the `QUIET_MAKE` setting. This seems to be a problem with `FCOMMON_OPT`. Setting `FFLAGS` seems to work just as well and does not result in these options being dropped. Have switched the PR to use `FFLAGS` and dropped `-fPIC` as that seemed to be set correctly anyways.